From 3729b131ef1dcaa043242e8074418249695d381b Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jirislaby@gmail.com>
Date: Sun, 24 Jun 2012 20:51:52 +0200
Subject: [PATCH] use grub2 as a package name

This will ease all of the renaming of directories and all the pkgdata
hacks.

Signed-off-by: Jiri Slaby <jirislaby@gmail.com>
---
 configure    |   24 ++++++++++++------------
 configure.ac |    2 +-
 2 files changed, 13 insertions(+), 13 deletions(-)

Index: grub-2.02/configure.ac
===================================================================
--- grub-2.02.orig/configure.ac
+++ grub-2.02/configure.ac
@@ -31,7 +31,7 @@ dnl (such as BUILD_CC, BUILD_CFLAGS, etc
 dnl with the prefix "TARGET_" (such as TARGET_CC, TARGET_CFLAGS, etc.) are
 dnl used for the target type. See INSTALL for full list of variables.
 
-AC_INIT([GRUB],[2.02],[bug-grub@gnu.org])
+AC_INIT([GRUB2],[2.02],[bug-grub@gnu.org])
 
 AC_CONFIG_AUX_DIR([build-aux])
 
